bin/varnishtest/tests/m00017.vtc

```diff
@@ -219,13 +217,21 @@ varnish v1 -vcl+backend {
 	}
 }
 
+server s1 -repeat 1 {
```

Comment:
Very minor, I think `-repeat 1` has no effect.

Reply:
it is required, server keeps the parameters across calls. I had a quick look if it made sense to change that (init with every call), but then we would get into the nitty gritty details of differentiating between the run, start or nothing and I thought it was not worth it.
So for now the repeat 1 basically resets the `repeat 5` from before.

Reply:
Please add an XXX comment before line 220 so we can forget about it for now.